kb/licensing/how-is-mbedtls-protected.md

```diff
@@ -2,18 +2,12 @@
 
 ## Copyright
 
-The copyright in the source code primarily protects Mbed TLS (formerly known as PolarSSL). We own all the copyright in the source code. For code that is contributed to us, we request the original author to share the copyright with us.
+The copyright in the source code primarily protects Mbed TLS (formerly known as PolarSSL). We own all the copyright in the source code. For code that is contributed to us, we request the original author to grant copyright to The Mbed TLS Contributors.
```

Comment:
I think "grant copyright" is wrong. The original author(s) retain the copyright, but the DCO means the author becomes one of "The Mbed TLS Contributors" and agrees to it being licensed under the licence we use (Apache 2.0). I made a suggested edit which you can see on the Conversation tab (as the text here has now changed)

Reply:
I think you're right, grant might be the wrong word to use. It's more about how we record the copyright - that we require that the copyright be recorded as "Mbed TLS Contributors" rather than naming specific individuals.